File: Backend/DTOs/Challenge/ChallengeActiveDTO.cs
namespace Learnz.DTOs;
public class ChallengeActiveDTO
{
    public string Name { get; set; }
    public List<ChallengePlayerResultDTO> Result { get; set; }
    public bool Cancelled { get; set; }
    public bool IsOwner { get; set; }
    public GeneralQuestionQuestionDTO? Question { get; set; }
    public int? LastQuestionPoint { get; set; }
    public ChallengeState State { get; set; }
}

File: Frontend/src/app/DTOs/Challenge/ChallengeActiveDTO.ts
import { ChallengeState } from "src/app/Enums/ChallengeState";
import { ChallengePlayerResultDTO } from "./ChallengePlayerResultDTO";
import { ChallengeQuestionDTO } from "./ChallengeQuestionDTO";

export interface ChallengeActiveDTO {
    name: string;
    result: ChallengePlayerResultDTO[];
    cancelled: boolean;
    isOwner: boolean;
    question: ChallengeQuestionDTO | null;
    lastQuestionPoint: number | null;
    state: ChallengeState;
}